tasks/queue-granules/schemas/input.json

```diff
@@ -2,68 +2,31 @@
   "title": "QueueGranulesInput",
   "description": "Describes the input and config used by the queue-granules task",
   "type": "object",
-  "required": [
-    "granules"
-  ],
+  "require": [ "granules" ],
   "properties": {
     "pdr": {
       "type": "object",
-      "required": [
-        "name",
-        "path"
-      ],
+      "required": ["name", "path"],
       "properties": {
-        "name": {
-          "type": "string"
-        },
-        "path": {
-          "type": "string"
-        }
+        "name": { "type": "string" },
+        "path": { "type": "string" }
       }
     },
     "granules": {
       "type": "array",
       "items": {
         "type": "object",
-        "required": [
-          "files",
-          "granuleId"
-        ],
+        "required": ["files", "granuleId"],
```

Comment:
Should files be required?    Looking through the code, it doesn't appear anything is referencing `input.granule.files`.

Reply:
Great call out, https://github.com/nasa/cumulus/pull/3441/commits/b0be488e8982321005f848642b7b91e5193f54ff